client/src/components/InfoCard/InfoCard.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./InfoCard.css";
+
+const Card = props => {
+  let { size = "large", color, picture } = props;
+  return (
+    <div className={`info-card-container info-card-${color} info-card-${size}`}>
+      {picture ? <img src={picture} alt="whoops"></img> : null}
```

Comment:
Could we have the `alt` text passed in as a prop?

Reply:
Changed file names and component name to card